chunkset_tpl.h

```diff
@@ -176,7 +176,18 @@ Z_INTERNAL uint8_t* CHUNKMEMSET(uint8_t *out, unsigned dist, unsigned len) {
 }
 
 Z_INTERNAL uint8_t* CHUNKMEMSET_SAFE(uint8_t *out, unsigned dist, unsigned len, unsigned left) {
+#ifdef UNALIGNED64_OK
+    static const uint32_t align_mask = 3;
```

Comment:
When `UNALIGNED64_OK` is defined, no mask should be needed.

Reply:
I was getting odd addresses like 0x7fbc8800946d, thus I though it would also hang in chunkmemset_4 and chunkmemset_2.

Reply:
Odd addresses are totally normal on little-endian machines, most big-endian machines need lower bits to be zero.